docs/source/json-api/index.rst

```diff
@@ -266,12 +266,36 @@ The JSON API essentially performs two separate tasks:
 
 .. note:: By default, the Daml Sandbox does not does not require access tokens. However, you still need to provide a party-specific access token when submitting commands or queries as a party. The token will not be validated in this case but it will be decoded to extract information like the party submitting the command.
 
-Party-specific Access Tokens
-----------------------------
+Party-specific Requests
```

Comment:
Not a big fan of this header, but I don't have any suggestions, and looks good otherwise.

Reply:
It seemed like a good fit for the paragraph (since it's the subject in the very first sentence) and I believe it lowers the possibility of somehow confusing this paragraph to something more closely related to something specific to custom claims access tokens. If by any chance you get a better idea, I'm open to making changes of course.